heaps/heap_sort.py

```diff
@@ -1,8 +1,21 @@
+from heapq import heappush, heappop
 
 
 def heap_sort(list):
```

Comment:
🤠 Nice implementation with `heapq`. How might you implement this with the `MinHeap` class you implemented below?